learning-bitmark/working-with-bitmarks/using-bitmark-shares.md

```diff
@@ -0,0 +1,631 @@
+  ```
+  ```json
+    {
+      "grantId": "fc88099d56c6f33bd90eb35937aa5d6e875c00017e7a78046ab3e5279b337e57",
+      "payId": "3728cb6a24f92801ac5fb0f29db58803339fecbe77dafad5cda9b1be9755a9a7d16b67428f7c3cebaccd10a20f6d8462",
+      "payments": {
+        "BTC": [
+          {
+            "currency": "BTC",
+            "address": "2MvrbNpfr6zWG2Mm2hnCh735hPwjoist3Ep",
+            "amount": "10000"
+          },
+          {
+            "currency": "BTC",
+            "address": "mr8DEygRvQwKfP4sVZuHVozqvzW89e193j",
+            "amount": "10000"
+          }
+        ],
+        "LTC": [
+          {
+            "currency": "LTC",
+            "address": "mv7pHLntNcEe8YQqCJGWTyL6aa8hjMRL9B",
+            "amount": "100000"
+          },
+          {
+            "currency": "LTC",
+            "address": "mzkCaHJmu1gdnsL9jxW2bwqtw2MCCy66Ds",
+            "amount": "100000"
+          }
+        ]
+      }
+    }
+  ```
+
+<br>
+
+* Pay for the grant shares transaction by LTC
+
+  ```shell
+    $ bitmark-wallet --conf ${XDG_CONFIG_HOME}/bitmark-wallet/test/test-bitmark-wallet.conf \
+      ltc --testnet \
+      sendmany --hex-data \
+      '3728cb6a24f92801ac5fb0f29db58803339fecbe77dafad5cda9b1be9755a9a7d16b67428f7c3cebaccd10a20f6d8462' \
+      'mv7pHLntNcEe8YQqCJGWTyL6aa8hjMRL9B,100000' \
+      'mzkCaHJmu1gdnsL9jxW2bwqtw2MCCy66Ds,100000'
+  ```
+  ```json
+    {
+      "txId": "f61c41e7261508f9f65fa73c5af93a86e02ba82361e8f01afde9245d0bbe82dc",
+      "rawTx": "0100000001b6e8...846200000000"
+    }
+  ```
+
+<br>
+
+* Verify the grant shares transaction status
+
+  ```shell
+    $ bitmark-cli -n testing \
+      status -t \
+      fc88099d56c6f33bd90eb35937aa5d6e875c00017e7a78046ab3e5279b337e57
+  ```
+  ```json
+    {
+      "status": "Verified"
+    }
+
+    //Check again after some minutes
+    {
+      "status": "Confirmed"
+    }
+  ```
+
+<br>
+
+* Check the sender's balance
+
+  ```shell
+    $ bitmark-cli -n testing balance -o first
+  ```
+  ```json
+    {
+      "balances": [
+        {
+          "shareId": "b069f2956b828281dec040782eea3d63793ab4cf17c26f7639e95f6f3b20ba23",
+          "confirmed": 900,
+          "spend": 0,
+          "available": 900
+        }
+      ]
+    }
+  ```
+
+<br>
+
+* Check the receiver's balance
+
+  ```shell
+    $ bitmark-cli -n testing balance -o second
+  ```
+  ```json
+    {
+      "balances": [
+        {
+          "shareId": "0f366f3ee477a75c6304746f45296d804c08be06ad0941313371a17fe07d778f",
+          "confirmed": 2000,
+          "spend": 0,
+          "available": 2000
+        },
+        {
+          "shareId": "b069f2956b828281dec040782eea3d63793ab4cf17c26f7639e95f6f3b20ba23",
+          "confirmed": 100,
+          "spend": 0,
+          "available": 100
+        }
+      ]
+    }
+  ```
+
+<br>
+<br>
+
+## Swapping bitmark shares
+
+Two owners of different bitmark shares are able to swap their shares together. The swap requires signatures from both owners.
+
```

Comment:
Much like the previous section, this could be neatly broken into subsections, with an overview summary appearing here.